### PR TITLE
[14.0] [MIG] server_environment_delivery: Migrate to version 14.0

### PR DESCRIPTION
**Pull request by sonhd91**

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 odoo-test-helper >= 1.1.0
 vcrpy-unittest
+odoo14-addon-server-environment @git+https://github.com/OCA/server-env@14.0#subdirectory=setup/server_environment
```
Comment (sebalix) [inline test-requirements.txt]: This one can already be removed, `server_environment` has been merged in 2020: https://github.com/OCA/server-env/pull/62

**Review by sebalix — COMMENTED** [review]
Small comment, LG

File: test-requirements.txt (hunk 1 above)
Comment (sonhd91) [reply test-requirements.txt]: Thank for remind, please check PR updated

**Review by sebalix — APPROVED** [review]

**Review by simahawk — APPROVED** [review]

**Review by mmequignon — APPROVED** [review]

**Pull request merged**